app/services/container_dashboard_service.rb

```diff
@@ -117,27 +119,41 @@ def get_url_to_entity(entity)
 
   def heatmaps
     # Get latest hourly rollup for each node.
-    node_ids = @ems.container_nodes if @ems
+    node_ids = @ems.container_nodes if @ems.present?
     metrics = MetricRollup.latest_rollups(ContainerNode.name, node_ids)
+    metrics = metrics.includes(:resource => [:ext_management_system]) unless @ems.present?
 
-    {
-      :nodeCpuUsage => metrics.collect { |m|
-        avg_cpu = m.cpu_usage_rate_average.round(2)
-        {
-          :id      => m.resource_id,
-          :tooltip => "#{m.resource.name} - #{avg_cpu}%",
-          :value   => avg_cpu / 100.0 # 1% should be 0.01
+    node_cpu_usage = []
+    node_memory_usage = []
+
+    if metrics.any?
+      metrics.each do |m|
+        node_cpu_usage << {
+          :id    => m.resource_id,
+          :info  => {
+            :node     => m.resource.name,
+            :provider => @ems.present? ? @ems.ext_management_system.name : m.resource.ext_management_system.name,
+            :total    => m.derived_vm_numvcpus
+          },
+          :value => (m.cpu_usage_rate_average / 100.0).round(CPU_USAGE_PRECISION)
```

Comment:
@simon3z 
Patternfly accepts the value here as a fraction, meaning 90% should be 0.9